### PR TITLE
Amount formatted by javascript with 2 digits

### PR DESCRIPTION
**Pull request by nickbristow**
On the new page, amount displays with 2 decimals now. Before only 1 was showing.

### CODE REVIEW

**Comment by nickbristow** [comment]
c2

**Comment by nickbristow** [comment]
https://trello.com/c/v4xgENRL/588-trailing-0-bug-in-price

File: app/assets/javascripts/details/views/details_request_card.js [hunk 1]
```diff
@@ -104,6 +104,20 @@ DetailsRequestCard = (function(){
...
+  DetailsRequestCard.prototype.formatData = function(data){
+    data.amount = this.formatMoney(data.amount, 2, '.', '');
+    return data;
+  }
+
+  DetailsRequestCard.prototype.formatMoney = function(num,c, d, t){
+    var n = num;
+    
+    var s = n < 0 ? "-" : "", 
+    i = parseInt(n = Math.abs(+n || 0).toFixed(c), 10) + "",
```
Comment (rememberlenny) [inline app/assets/javascripts/details/views/details_request_card.js]: Curious- is this okay as parseInt and not parseFloat?

File: app/views/ncr/fields/_amount.html.haml [hunk 2]
```diff
@@ -27,7 +27,7 @@
             = t(t_slug + "amount")
         .medium-8.column
           %span.detail-value{class: "amount" }
-            = client_data["amount"]
+            = number_with_precision(client_data["amount"], precision: 2)
```
Comment (rememberlenny) [inline app/views/ncr/fields/_amount.html.haml]: solid

**Pull request merged**